tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -71,10 +71,19 @@ cuda_py_test(
     ],
 )
 
+py_library(
+    name = "benchmark_util",
+    srcs = ["benchmark_util.py"],
+    deps = [
+        "//tensorflow:tensorflow_py",
+        "//third_party/py/numpy",
+    ],
+)
+
 py_test(
-    name = "keras_examples_benchmark_test",
+    name = "bidirectional_lstm_benchmark_test",
     size = "medium",
-    srcs = ["keras_examples_benchmark_test.py"],
+    srcs = ["keras_examples_benchmarks/bidirectional_lstm_benchmark_test.py"],
     python_version = "PY3",
     tags = [
         "no_pip",  # b/161253163
```

Comment:
no_windows as well?

Reply:
I think I already added, you can check[ line 90](https://github.com/tensorflow/tensorflow/pull/41249/commits/4ada0936240434bc9744db87f4a96c69617533bb#diff-186433d6a449914056aee3a2fc32e63cR90).

Reply:
Ah, it's folded. Thanks.